### PR TITLE
docs: Rename "master" organization to "host"

### PR DESCRIPTION
**Pull request by sebathomas**
## Summary

This was [changed](https://github.com/magma/magma/pull/11374) as part of the 1.8 release but not adapted in all parts of the documentation.

## Test Plan

n/a

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: docs/readmes/nms/deploy_setup.md [hunk 1]
```diff
@@ -20,22 +20,22 @@ would access the NMS using `facebook.nms.yourdomain.com`.
 ## First-time Setup
```
Comment (voisey) [inline docs/readmes/nms/deploy_setup.md:20]: It looks like the "master" on line 13 of this file was missed

File: docs/docusaurus/versioned_docs/version-1.8.0/orc8r/deploy_install.md [hunk 2]
```diff
@@ -262,11 +262,11 @@ also need to add a new admin user with the updated `admin_operator` cert.
...
-Create an admin user for the `master` organization on the NMS
+Create an admin user for the `host` organization on the NMS
 
 ```bash
 kubectl --namespace orc8r exec -it deploy/nms-magmalte -- \
-  yarn setAdminPassword master ADMIN_USER_EMAIL ADMIN_USER_PASSWORD
+  yarn setAdminPassword host ADMIN_USER_EMAIL ADMIN_USER_PASSWORD
 ```
 
 ## DNS Resolution
```
Comment (voisey) [inline docs/docusaurus/versioned_docs/version-1.8.0/orc8r/deploy_install.md:272]: It looks like two cases have been missed on lines 318, 319 of this file

**Review by voisey — COMMENTED** [review]
Nice find :slightly_smiling_face: A couple of things:

- I think some cases may have been missed from other files: [architecture_overview.md](https://github.com/magma/magma/blob/master/docs/readmes/nms/architecture_overview.md#masterorgmiddleware), [metrics.md](https://github.com/magma/magma/blob/master/docs/readmes/nms/metrics.md#enabling-access), [deploy_using_juju.md](https://github.com/magma/magma/blob/master/docs/readmes/orc8r/deploy_using_juju.md#verify-the-deployment)
- We may want to propagate all the changes you've made to the 1.8 docs, because as it stands some of the changes won't be reflected in them. I'm not sure whether there's an automated way to do this, but the manual way would be to copy the files to the respective versioned docs folders, edit the headers by changing the `id` and setting the `original_id`, and then running `make dev` to generate the changes to the `en.json`.

**Comment by sebathomas** [comment]
Good catch, thank you! I'm working on adapting all of this.

Do you know why metrics.md and a lot of NMS pages don't exist in the v1.8 version of the docs?

**Comment by voisey** [comment]
> Good catch, thank you! I'm working on adapting all of this.
> 
> Do you know why metrics.md and a lot of NMS pages don't exist in the v1.8 version of the docs?

I think that if there aren't any modifications to a certain file between versions, the file isn't copied across to the newer versioned docs and instead the file from the previous version is used. For instance, if I modify version-1.7.0/nms/metrics.md locally, which doesn't have a 1.8 version, I see the changes in the 1.8 docs.

**Review by voisey — APPROVED** [review]

**Pull request merged**